### ISSUE TITLE
DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0

### ISSUE THREAD

**Issue by MissedWarrior** [pierreburel/sass-rem#13]
Hey! Just updated the sass package and got this warning:
```
DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div($value, $rem-baseline)

More info and automated migrator: https://sass-lang.com/d/slash-div

   ╷
35 │       $result: append($result, $value / $rem-baseline * 1rem, $separator);
   │                                ^^^^^^^^^^^^^^^^^^^^^^
   ╵
    node_modules\sass-rem\_rem.scss 35:32            rem-convert()
    node_modules\sass-rem\_rem.scss 53:13            rem()
    src\assets\style\variables\typography.scss 3:13  @import
    src\assets\style\variables\index.scss 6:9        @import
    stdin 1:9                                        root stylesheet
```

Any plans to support Dart Sass 2.0.0?

**Comment by pierreburel** [comment]
Hello! I plan to support the new module system, but i need some time to find out how not to lose retrocompatibility with older versions of Sass… I'll try to do that before the release of Dart 2.0.
In the meantime, if you use PostCSS in your projects you can use https://github.com/pierreburel/postcss-rem

**Comment by pierreburel** [comment]
Should be fixed by 3.0, but it's a Dart Sass only release.
